packages/itwin/property-grid/src/PropertyGridUiItemsProvider.tsx

```diff
@@ -40,7 +40,10 @@ const onPresentationSelectionChanged = async (evt: SelectionChangeEventArgs, sel
       }
     );
     if (instanceKeys.some((key) => !Id64.isTransient(key.id))) {
-      widgetDef.setWidgetState(WidgetState.Open);
+      // if the widget is minimized, do not force it open
```

Comment:
could you comment this is for appui 1.0?

Reply:
if you look below this whole routine is in an if block for that
```typescript
if (UiFramework.uiVersion === "1") {
      this._removeListenerFunc = Presentation.selection.selectionChange.addListener(onPresentationSelectionChanged);
    }

Reply:
sorry didnt look at the larger diff. resolving

Reply:
i added a comment to the routine as well though so its more clear